src/Sound/Tidal/UI.hs

```diff
@@ -73,22 +96,22 @@ timeToRands' seed n
 
 {-|
 
-`rand` generates a continuous pattern of (pseudo-)random numbers between `0` and `1`.
+`rand` generates a continuous pattern of (pseudo-)random numbers between @0@ and @1@.
 
 @
 sound "bd*8" # pan rand
 @
 
-pans bass drums randomly
+pans bass drums randomly, and
 
 @
 sound "sn sn ~ sn" # gain rand
 @
 
-makes the snares' randomly loud and quiet.
+makes the snares randomly loud and quiet.
 
-Numbers coming from this pattern are 'seeded' by time. So if you reset
-time (via `cps (-1)`, then `cps 1.1` or whatever cps you want to
+Numbers coming from this pattern are \'seeded\' by time. So if you reset
```

Comment:
Consider referring to `setCycle` or `resetCycles` instead. I would be surprised if `cps (-1)` actually works.

Reply:
I tested out `cps (-1)` and it does indeed work. But I think it’s better, as you say, to mention `resetCycles` and `setCycle` first.